opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/EventAccumulator.java

```diff
@@ -61,6 +61,16 @@ public void sendNow(final Log log) {
         }
     }
 
+    @Override
+    public void sendNowSync(Event event) {
+        sendNowSync(event);
+    }
+
+    @Override
+    public void sendNowSync(Log eventLog) {
+        sendNowSync(eventLog);
```

Comment:
This is throwing a `StackOverflowException` if invoked. Should probably be `sendNow(event)`